<html>
<head>
<title>MayaChemTools: References</title>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<link rel="stylesheet" type="text/css" href="./css/MayaChemTools.css">
</head>
<body leftmargin="20" rightmargin="20" topmargin="10" bottommargin="10">
<br/>
<center>
<a href="http://www.mayachemtools.org" title="MayaChemTools Home"><img src="./images/MayaChemToolsLogo.gif" border="0" alt="MayaChemTools"></a>
</center>
<br/>
<h1>References</h1>
<div class="References">
<table width="100%" border="0" cellspacing="0" cellpadding="0">
<tr><td width="30" align="right" valign="top">1.</td><td width="5">&nbsp;</td><td align="left" valign="top">Wall, L.; Christiansen, T.; Schwartz, R.L. Programming Perl, 2nd edition. O'Reilly Media Inc., September 1996.</td></tr>
<tr><td align="right" valign="top">2.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.cpan.org" title="Visit http://www.cpan.org">CPAN: Comprehensive Perl archive network.</a></td></tr>
<tr><td align="right" valign="top">3.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.fsf.org" title="Visit http://www.fsf.org">FSF: Free software foundation.</a></td></tr>
<tr><td align="right" valign="top">4.</td><td>&nbsp;</td><td align="left" valign="top">Knuth, D.E. The art of computer programming. Vol. 1-3. 2nd edition. Addison-Wesley, September 1998.</td></tr>
<tr><td align="right" valign="top">5.</td><td>&nbsp;</td><td align="left" valign="top">Press, W.H.; Teukolsky, S.A.; Vetterling, W.T.; Flannery, B.P. Numerical recipies in C: the art of scientific computing. 2nd edition. Cambridge University Press, 1992.</td></tr>
<tr><td align="right" valign="top">6.</td><td>&nbsp;</td><td align="left" valign="top">Orwant, J.; MacDonald, J.; Hietaniemi, J. Mastering algorithms with Perl. O'Reilly Media Inc., August 1999.</td></tr>
<tr><td align="right" valign="top">7.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.webelements.com" title="Visit http://www.webelements.com">Data for elements in the periodic table.</a></td></tr>
<tr><td align="right" valign="top">8.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://physics.nist.gov/PhysRefData/Compositions/index.html" title="Visit http://physics.nist.gov/PhysRefData/Compositions/index.html">Isotope data for elements in the periodic table.</a></td></tr>
<tr><td align="right" valign="top">9.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.expasy.ch" title="Visit http://www.expasy.ch">Main data source for amino acids.</a></td></tr>
<tr><td align="right" valign="top">10.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.perlmol.org" title="Visit http://www.perlmol.org">PerlMol - Perl modules for molecular chemistry.</a></td></tr>
<tr><td align="right" valign="top">11.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://openbabel.sourceforge.net/wiki/Main_Page" title="Visit http://openbabel.sourceforge.net/wiki/Main_Page">OpenBabel: The open source chemistry toolbox.</a></td></tr>
<tr><td align="right" valign="top">12.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://cdk.sourceforge.net" title="Visit http://cdk.sourceforge.net">CDK: The chemistry development kit.</a></td></tr>
<tr><td align="right" valign="top">13.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://sourceforge.net/projects/joelib/" title="Visit http://sourceforge.net/projects/joelib/">JOELIB.</a></td></tr>
<tr><td align="right" valign="top">14.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://accelrys.com/products/informatics/cheminformatics/ctfile-formats/no-fee.php" title="Visit http://accelrys.com/products/informatics/cheminformatics/ctfile-formats/no-fee.php">CTFile Formats.</a></td></tr>
<tr><td align="right" valign="top">15.</td><td>&nbsp;</td><td align="left" valign="top">Conway, D. Object oriented Perl. 1st edition. O'Reilly Media Inc., January 2000.</td></tr>
<tr><td align="right" valign="top">16.</td><td>&nbsp;</td><td align="left" valign="top">Friedl, J.E.F. Mastering regular expressions. 3rd edition. O'Reilly Media Inc., August 2006.</td></tr>
<tr><td align="right" valign="top">17.</td><td>&nbsp;</td><td align="left" valign="top">Schulz, G.E.; Schirmer, R.H. Principles of protein structure. Springer-Verlag, January 1997.</td></tr>
<tr><td align="right" valign="top">18.</td><td>&nbsp;</td><td align="left" valign="top">Saenger, W. Principles of nucleic acid structure. Springer-Verlag, 1983.</td></tr>
<tr><td align="right" valign="top">19.</td><td>&nbsp;</td><td align="left" valign="top">Cornish-Bowden, A. Nomenclature for incompletely specified bases in nucleic acid sequence. Nucleic Acids Res. 1985, 13, 3021-3030.</td></tr>
<tr><td align="right" valign="top">20.</td><td>&nbsp;</td><td align="left" valign="top">Clapham, C. A concise Oxford dictionary of mathematics. Oxford University Press, 1990.</td></tr>
<tr><td align="right" valign="top">21.</td><td>&nbsp;</td><td align="left" valign="top">Cook, J.L. Conversion factors. Oxford University Press, 1993.</td></tr>
<tr><td align="right" valign="top">22.</td><td>&nbsp;</td><td align="left" valign="top">Pauling, L. The nature of chemical bond. 3rd edition. Cornell University Press, June 1960.</td></tr>
<tr><td align="right" valign="top">23.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.daylight.com/dayhtml/doc/theory/index.html" title="Visit http://www.daylight.com/dayhtml/doc/theory/index.html">Daylight theory manual.</a></td></tr>
<tr><td align="right" valign="top">24.</td><td>&nbsp;</td><td align="left" valign="top">Weininger, D. SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules. J. Am. Chem. Soc. 1988, 28, 31-36.</td></tr>
<tr><td align="right" valign="top">25.</td><td>&nbsp;</td><td align="left" valign="top">Weininger, D.; Weininger, A.; Weininger, J.L. SMILES. 2. Algorithm for generation of unique SMILES notation. J. Am. Chem. Soc. 1989, 29, 97-101.</td></tr>
<tr><td align="right" valign="top">26.</td><td>&nbsp;</td><td align="left" valign="top">Weininger, D. SMILES. 3. Depit. Graphical depiction of chemical structures. J. Am. Chem. Soc. 1990, 30, 237-243.</td></tr>
<tr><td align="right" valign="top">27.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://eyesopen.com/docs/toolkits/current/pdf/OEChem_TK-c++.pdf" title="Visit http://eyesopen.com/docs/toolkits/current/pdf/OEChem_TK-c++.pdf">OEChem TK manual.</a></td></tr>
<tr><td align="right" valign="top">28.</td><td>&nbsp;</td><td align="left" valign="top">Parkin, G. Valence, oxidation number, and formal charge: Three related but fundamentally different concepts. J. Chem. Educ. 2006, 83, 791-799.</td></tr>
<tr><td align="right" valign="top">29.</td><td>&nbsp;</td><td align="left" valign="top">Gateiger, J.; Jochum, C. An algorithm for the perception of synthetically important rigngs. J. Chem. Inf. Comput. Sci. 1979, 19, 43-47.</td></tr>
<tr><td align="right" valign="top">30.</td><td>&nbsp;</td><td align="left" valign="top">Balducci, R.; Pearlman, R.S. Efficient exact solution of the ring perception problem. J. Chem. Inf. Comput. Sci. 1994, 34, 822-831.</td></tr>
<tr><td align="right" valign="top">31.</td><td>&nbsp;</td><td align="left" valign="top">Hanser, T.; Jauffret, P.; Kaufmann, G. A new algorithm for exhaustive ring perception in a molecular graph. J. Chem. Inf. Comput. Sci. 1996, 36, 1146-1152.</td></tr>
<tr><td align="right" valign="top">32.</td><td>&nbsp;</td><td align="left" valign="top">Cahn, R.S.; Ingold, C.; Prelog, V. Specification of molecular chirality. Angew. Chem. Internat. Edit. 1966, 5, 385-415.</td></tr>
<tr><td align="right" valign="top">33.</td><td>&nbsp;</td><td align="left" valign="top">Prelog, V.; Helmchen, G. Basic principles of the CIP-system and proposals for revision. Angew. Chem. Internat. Edit. 1982, 21, 567-583.</td></tr>
<tr><td align="right" valign="top">34.</td><td>&nbsp;</td><td align="left" valign="top">Mata, P.; Lobo, A.M.; Marshall, C.; Johnson, P.A. The CIP seqeunce rules: Analysis and proposal for a revision. Tetrahedron. 1993, 4, 657-668.</td></tr>
<tr><td align="right" valign="top">35.</td><td>&nbsp;</td><td align="left" valign="top">Nourse, J.G.; Carhart, R.E.; Smith, D.H.; Djerassi, C. Exhaustive generation of stereoisomers for structure elucidation. J. Am. Chem. Soc. 1979, 101, 1216-1223.</td></tr>
<tr><td align="right" valign="top">36.</td><td>&nbsp;</td><td align="left" valign="top">Nourse, J.G.; Smith, D.H.; Carhart, R.E.; Djerassi, C. Computer-assisted elucidation of molecular structue with stereochemistry. J. Am. Chem. Soc. 1980, 102, 6289-6295.</td></tr>
<tr><td align="right" valign="top">37.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.chem.qmul.ac.uk/iupac/fusedring/" title="Visit http://www.chem.qmul.ac.uk/iupac/fusedring/">Fused ring systems.</a></td></tr>
<tr><td align="right" valign="top">38.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.burtleburtle.net/bob/hash/doobs.html" title="Visit http://www.burtleburtle.net/bob/hash/doobs.html">A hash function for hash table lookup.</a></td></tr>
<tr><td align="right" valign="top">39.</td><td>&nbsp;</td><td align="left" valign="top">Ralaivola, L.; Swamidass, S.J.; Saigo, H.; Baldi, P. Graph kernals for chemical informatics. Neural Networks. 2005, 18, 1093-1110.</td></tr>
<tr><td align="right" valign="top">40.</td><td>&nbsp;</td><td align="left" valign="top">Willett. P.; Barnard, J.M.; Downs, G.M. Chemical Similarity Searching. J. Chem. Inf. Comput. Sci. 1998, 38, 983-996.</td></tr>
<tr><td align="right" valign="top">41.</td><td>&nbsp;</td><td align="left" valign="top">Holliday, J.D.; Hu, C-Y.; Willett, P. Grouping of coefficients for the calculation of inter-molecular similarity and dissimilarity using 2D fragment bit-strings, Combinatorial Chemistry & High Throughput Screening. 2002, Vol. 5, No. 2, 155-166.</td></tr>
<tr><td align="right" valign="top">42.</td><td>&nbsp;</td><td align="left" valign="top">Flinger, M.; Verducci, J.; Blower, P. A modification of the Jacard-Tanimoto similarity index for diverse selection of chemical compounds using binary strings. Technometrics. 2002, 44, 110-119.</td></tr>
<tr><td align="right" valign="top">43.</td><td>&nbsp;</td><td align="left" valign="top">Wang, Y.; Bajorath, J. Balancing the influence of molecular complexity in fingerprint similarity searching. J. Chem. Inf. Comput. Sci. 2008, 48, 75-84.</td></tr>
<tr><td align="right" valign="top">44.</td><td>&nbsp;</td><td align="left" valign="top">Flower, D.R. On the properties of bit string-based measures of chemical similarity. J. Chem. Inf. Comput. Sci. 1998, 38, 379-386.</td></tr>
<tr><td align="right" valign="top">45.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://accelrys.com/products/pdf/keys-to-keyset-technology.pdf" title="Visit http://accelrys.com/products/pdf/keys-to-keyset-technology.pdf">The Enkfil.dat and Eksfil.dat files: The keys to understanding MDL keyset technology.</a></td></tr>
<tr><td align="right" valign="top">46.</td><td>&nbsp;</td><td align="left" valign="top">Durant, J.L.; Leland, B.A.; Henry, D.H.; Nourse, J.G. Reoptimization of MDL Keys for Use in Drug Discovery. J. Chem. Inf. Comput. Sci. 2002, 42, 1273-1280.</td></tr>
<tr><td align="right" valign="top">47.</td><td>&nbsp;</td><td align="left" valign="top"><a href="https://list.indiana.edu/sympa/arc/chminf-l/2007-11/msg00058.html" title="Visit https://list.indiana.edu/sympa/arc/chminf-l/2007-11/msg00058.html">Description of public MACCS keys.</a></td></tr>
<tr><td align="right" valign="top">48.</td><td>&nbsp;</td><td align="left" valign="top">Morgan, H.L. The generation of a unique machine description for chemical structures - A technique developed at chemical abstracts service. J. Chem. Doc. 1965, 5, 107-112.</td></tr>
<tr><td align="right" valign="top">49.</td><td>&nbsp;</td><td align="left" valign="top">Penny, R.H. A connectivity code for use in describing chemical structures. J. Chem. Doc. 1965, 5, 113-117. J. Chem. Doc. 1973, 3, 153-157.</td></tr>
<tr><td align="right" valign="top">50.</td><td>&nbsp;</td><td align="left" valign="top">Adamson, G.W.; Cowell, J.; Lynch, M.F.; McLure, A.H.; Town, W.G. Yapp, M. Strategic considerations in design of a screening system for substructure searches of chemical structure files.</td></tr>
<tr><td align="right" valign="top">51.</td><td>&nbsp;</td><td align="left" valign="top">Wipke, W.T.; Krishnan, S.; Ouchi, G.I. Hash functions for rapid storage and retrieval of chemical structures. J. Chem. Inf. Comput. Sci. 2002, 42, 1273-1280. 1978, 18, 31- .</td></tr>
<tr><td align="right" valign="top">52.</td><td>&nbsp;</td><td align="left" valign="top">Rogers, D.; Hahn, M. Extended-connectivity fingerprints. J. Chem. Inf. Mod. 2010, 50, 742-754.</td></tr>
<tr><td align="right" valign="top">53.</td><td>&nbsp;</td><td align="left" valign="top">Faulon, J.-L.; Visco, D.P., Jr.; Pophale, R.S. The Signature Molecular Descriptor. 1. Using extended valence sequences in QSAR and QSPR studies. J. Chem. Inf. Comput. Sci. 2003, 43, 707-720.</td></tr>
<tr><td align="right" valign="top">54.</td><td>&nbsp;</td><td align="left" valign="top">Faulon, J.-L.; Collins, M.J.; Carr, R.D. The signature molecular descriptor. 4. Canonizing molecules using extended valence sequences. J. Chem. Inf. Comput. Sci. 2004, 44, 427-436.</td></tr>
<tr><td align="right" valign="top">55.</td><td>&nbsp;</td><td align="left" valign="top">Bender, A.; Mussa, H.Y.; Glen, R.C.; Reiling, S. Molecular similarity searching using atom environments, information-based feature selection, and a naive bayesian classifier. J. Chem. Inf. Comput. Sci. 2004, 44, 170-178.</td></tr>
<tr><td align="right" valign="top">56.</td><td>&nbsp;</td><td align="left" valign="top">Bender, A.; Mussa, H.Y.; Glen, R.C.; Reiling, S. Similarity searching of chemical databases using atom environment descriptors (MOLPRINT 2D): Evaluation of performance. J. Chem. Inf. Comput. Sci. 2004, 44, 1708-1718.</td></tr>
<tr><td align="right" valign="top">57.</td><td>&nbsp;</td><td align="left" valign="top">Carhart, R.E.; Smith, D.H.; Venkataraghavan, R. Atom pairs as molecular features in structure-activity studies: Definition and application. J. Chem. Inf. Comput. Sci. 1985, 25, 64-73.</td></tr>
<tr><td align="right" valign="top">58.</td><td>&nbsp;</td><td align="left" valign="top">Nilakantan, R.; Bauman, N.; Dixon, J.S.; Venkataraghavan, R. Topological torsion: A new molecular descriptor for SAR applications. Comparison with other descriptors. J. Chem. Inf. Comput. Sci. 1987, 27, 82-85.</td></tr>
<tr><td align="right" valign="top">59.</td><td>&nbsp;</td><td align="left" valign="top">Langham, J.L.; Jain, A.N. Accurate and interpretable computational modeling of chemical mutagenicity.  J. Chem. Inf. Comput. Sci. 2008, 48, 1833-1839.</td></tr>
<tr><td align="right" valign="top">60.</td><td>&nbsp;</td><td align="left" valign="top">Schneider, G.; Neidhart, W.; Giller, T.; Schmid, G. Scaffold-hopping by topological pharmacophore search: A contribution to virtual screening. Angew. Chem. Int. Ed. 1999, 38, 2894-2896.</td></tr>
<tr><td align="right" valign="top">61.</td><td>&nbsp;</td><td align="left" valign="top">Fechner, U.; Franke, L.; Renner, S.; Schneider, P. Schneider, G. Comparison of correlation vector methods for ligand-based similarity searching. J. Comput. Aided Mol. Des. 2003, 17, 687-698.</td></tr>
<tr><td align="right" valign="top">62.</td><td>&nbsp;</td><td align="left" valign="top">Fechner, U.; Schneider, G. Evaluation of distance metrics for ligand-based similarity searching. ChemBioChem. 2004, 5, 538-540.</td></tr>
<tr><td align="right" valign="top">63.</td><td>&nbsp;</td><td align="left" valign="top">Downs, G.M.; Willett, P.; Fisanick, W. Similarity searching and clustering of chemical-structure databases using molecular property data.  J. Chem. Inf. Comput. Sci., 1994, 34, 1094-1102.</td></tr>
<tr><td align="right" valign="top">64.</td><td>&nbsp;</td><td align="left" valign="top">Chen, X.; Reynolds, C.H.; Performance of similarity measures in 2D fragment-based similarity searching: Comparison of structural descriptors and similarity coefficients. J. Chem. Inf. Comput. Sci. 2002, 42, 1407-1414.</td></tr>
<tr><td align="right" valign="top">65.</td><td>&nbsp;</td><td align="left" valign="top">Steffen, R.; Fechner, U.; Schneider, G. Alignment-free pharmacophore patterns: A correlation-vector approach. Pharmacophores and pharmacophore searches. 2006. Volume 32. Wiley-VCH. 49-80.</td></tr>
<tr><td align="right" valign="top">66.</td><td>&nbsp;</td><td align="left" valign="top">McGregor, M.J.; Muskal, S. M. Pharmacophore fingerprinting. 1. Application to QSAR and focused library design.  J. Chem. Inf. Comput. Sci. 1999, 39, 569-574.</td></tr>
<tr><td align="right" valign="top">67.</td><td>&nbsp;</td><td align="left" valign="top">Floyd, R.W. Algorithm 97: Shortest path. Communications of the ACM. 1962, 5, 345.</td></tr>
<tr><td align="right" valign="top">68.</td><td>&nbsp;</td><td align="left" valign="top">Horvath, D. Topological pharmacophores. Cheminformatics approaches to virtual screening. 2008. RSC Publishing. 44-75.</td></tr>
<tr><td align="right" valign="top">69.</td><td>&nbsp;</td><td align="left" valign="top">Ewing, T.; Baber, C.; Feher, M. Novel 2D fingerprints in ligand-based virtual screening. J. Chem. Inf. Model. 2006, 46, 2423-2431.</td></tr>
<tr><td align="right" valign="top">70.</td><td>&nbsp;</td><td align="left" valign="top">Watson, P. Naive Bayes classification using 2D pharmacophore feature triplet vectors. J. Chem. Inf. Model. 2008, 48, 166-178</td></tr>
<tr><td align="right" valign="top">71.</td><td>&nbsp;</td><td align="left" valign="top">Bonachera, F.; Parent, B.; Barbosa, F.; Froloff, N.; Horvath, D. Fuzzy tricentric pharmacophore fingerprints. 1. Topological fuzzy pharmacophore triplets and adapted molecular similarity scoring schemes. J. Chem. Inf. Model., 2006, 46, 2457-2477.</td></tr>
<tr><td align="right" valign="top">72.</td><td>&nbsp;</td><td align="left" valign="top">Kearsley, S.K.; Sallamack, S.; Fluder, E.M.; Andose, J.D.; Mosley, R.T.; Sheridan, R.P. Chemical Similarity Using Physiochemical Property Descriptors.J. Chem. Inf. Comput. Sci., 1996, 36, 118-127.</td></tr>
<tr><td align="right" valign="top">73.</td><td>&nbsp;</td><td align="left" valign="top">Filimonov, D.; Poroikov, V.; Borodina, Y.; Gloriozova, T. Chemical similarity assessment through multilevel neighborhoods of atoms:  Definition and comparison with the other Descriptors. J. Chem. Inf. Comput. Sci., 1999, 39, 666-670.</td></tr>
<tr><td align="right" valign="top">74.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.rdkit.org" title="Visit http://www.rdkit.org">RDKit - Cheminformatics and Machine Learning Software.</a></td></tr>
<tr><td align="right" valign="top">75.</td><td>&nbsp;</td><td align="left" valign="top">Kier, L.B.; Hall, L.H. Electrotopological state indices for atom types: A novel combination of electronic, topological, and valence state information. J. Chem. Inf. Comput. Sci. 1995, 35, 1039-1045.</td></tr>
<tr><td align="right" valign="top">76.</td><td>&nbsp;</td><td align="left" valign="top">Kier, L.B.; Hall, L.H. Molecular structure description - The electrotopological state. Academic Press, 1999.</td></tr>
<tr><td align="right" valign="top">77.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.edusoft-lc.com/molconn/" title="Visit http://www.edusoft-lc.com/molconn/">Molconn-Z - Program for generation of Molecular Connectivity, Shape, and Information Indices.</a></td></tr>
<tr><td align="right" valign="top">78.</td><td>&nbsp;</td><td align="left" valign="top">Kier, L.B.; Hall, L.H. The E-State as the basis for molecular structure space definition and structure similarity. J. Chem. Inf. Comput. Sci. 2000, 40, 784-791.</td></tr>
<tr><td align="right" valign="top">79.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.tripos.com/tripos_resources/fileroot/pdfs/mol2_format2.pdf" title="Visit http://www.tripos.com/tripos_resources/fileroot/pdfs/mol2_format2.pdf">SYBYL atom types.</a></td></tr>
<tr><td align="right" valign="top">80.</td><td>&nbsp;</td><td align="left" valign="top">Clark, M.; Cramer III, R.D.; Opdenbosch, N.V. Validation of the general purpose Tripos 5.2 forcefield. J. Comput. Chem. 1989, 10, 982-1012.</td></tr>
<tr><td align="right" valign="top">81.</td><td>&nbsp;</td><td align="left" valign="top">Rappe, A.K.; Casewit, C.J.; Colwell, K.S.; Goddard III, W.A.; Skiff, W.M. UFF, a full periodic table force field for molecular mechanics and molecular dynamics simulations. J. Am. Chem. Soc. 1992, 114, 10024-10035.</td></tr>
<tr><td align="right" valign="top">82.</td><td>&nbsp;</td><td align="left" valign="top">Rappe, A. K. Personal communication. 2009.</td></tr>
<tr><td align="right" valign="top">83.</td><td>&nbsp;</td><td align="left" valign="top">Halgren, T.A.; Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94. 1996, J. Comput. Chem., 17, 490-519.</td></tr>
<tr><td align="right" valign="top">84.</td><td>&nbsp;</td><td align="left" valign="top">Halgren, T.A.; Merck molecular force field. II. MMFF94 van der Waals and electrostatic parameters for intermolecular interactions. J. Compt. Chem. 1996, 17, 520-552.</td></tr>
<tr><td align="right" valign="top">85.</td><td>&nbsp;</td><td align="left" valign="top">Halgren, T.A.; Merck molecular force field. III. Molecular geometries and vibrational frequencies for MMFF94. J. Compt. Chem. 1996, 17,  553-586.</td></tr>
<tr><td align="right" valign="top">86.</td><td>&nbsp;</td><td align="left" valign="top">Halgren, T.A.; Nachbar, R. B.; Merck molecular force field. IV. conformational energies and geometries for MMFF94. J. Compt. Chem. 1996, 17, 587-615.</td></tr>
<tr><td align="right" valign="top">87.</td><td>&nbsp;</td><td align="left" valign="top">Halgren, T.A.; Merck molecular force field. V. Extension of MMFF94 using experimental data, additional computational data, and empirical rules. J. Compt. Chem. 1996, 17, 616-641.</td></tr>
<tr><td align="right" valign="top">88.</td><td>&nbsp;</td><td align="left" valign="top">Mayo, S.L.; Olafson, B.A.; Goddard III, W.A. DREIDING: A Generic Force Field for Molecular Simulations. J. Phys. Chem. 1990, 94, 8897-8909.</td></tr>
<tr><td align="right" valign="top">89.</td><td>&nbsp;</td><td align="left" valign="top">Wildman, S.A.; Crippen, G.M.; Prediction of Physicochemical Parameters by Atomic Contributions. J. Chem. Inf. Comput. Sci. 1999, 39, 868-873.</td></tr>
<tr><td align="right" valign="top">90.</td><td>&nbsp;</td><td align="left" valign="top">Ertl, P.; Rohde, B.; Selzer, P. Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport Properties. J. Med. Chem. 2000, 43, 3714-3717.</td></tr>
<tr><td align="right" valign="top">91.</td><td>&nbsp;</td><td align="left" valign="top">Ertl, P. Personal communication. 2010.</td></tr>
<tr><td align="right" valign="top">92.</td><td>&nbsp;</td><td align="left" valign="top">Veber, D.F.; Johnson, S. R.; Chend, H.Y.; Smith, B.R.; Ward, K.W.; Kopple, K.D. Molecular properties that influence the oral bioavailability of drug candidates. J. Med. Chem. 2002, 45, 2165-2623.</td></tr>
<tr><td align="right" valign="top">91.</td><td>&nbsp;</td><td align="left" valign="top">Lipinski, C.A.; Lombardo, F.; Dominy, B.W.; Feeney, P.J. Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. Adv. Drug. Del. Rev. 1997, 23, 3-25.</td></tr>
<tr><td align="right" valign="top">92.</td><td>&nbsp;</td><td align="left" valign="top">Congreve M.; Carr R., Murray C., Jhoti H.A. 'rule of three' for fragment-based lead discovery? Drug. Discov. Today. 2003, 8, 876-877.</td></tr>
<tr><td align="right" valign="top">93.</td><td>&nbsp;</td><td align="left" valign="top">Zhao, Y.H.; Abraham, M.H.; Zissimos, A.M. Fast calculation of van der Waals volume as a sum of atomic and bond contributions and its application to drug compounds. J. Org. Chem. 2003, 68, 7368-7373.</td></tr>
<tr><td align="right" valign="top">94.</td><td>&nbsp;</td><td align="left" valign="top">Chen, J.; Holliday, J.; Bradshaw, J.A machine learning approach to weighting schemes in the data fusion of similarity coefficients. J. Chem. Inf. Model. 2009, 49, 185-194.</td></tr>
<tr><td align="right" valign="top">95.</td><td>&nbsp;</td><td align="left" valign="top">Williams, C. Reverse fingerprinting, similarity searching by group fusion and fingerprint bit importance. Molecular Diversity. 2006, 10, 311-332.</td></tr>
<tr><td align="right" valign="top">96.</td><td>&nbsp;</td><td align="left" valign="top">Whittle, M.; Gillet, V.J.; Willett, P.; Loesel, J. Analysis of data fusion methods in virtual screening: Similarity and group Fusion. J. Chem. Inf. Model. 2006, 46, 2206-2219.</td></tr>
<tr><td align="right" valign="top">97.</td><td>&nbsp;</td><td align="left" valign="top">Hert, J.; Willett, P.; Wilton, D.J.; Acklin, P.; Azzaoui, K.; Jacoby, E.; Schuffenhauer, A. New methods for ligand-based virtual screening: Use of data fusion and machine learning to enhance the effectiveness of similarity searching. J. Chem. Inf. Model. 2006, 46, 462-470.</td></tr>
<tr><td align="right" valign="top">98.</td><td>&nbsp;</td><td align="left" valign="top">Chu, C-W.; Holliday, J.D.; Willett, P. Effect of data standardization on chemical clustering and similarity searching. J. Chem. Inf. Model., 2009, 49, 155-161.</td></tr>
<tr><td align="right" valign="top">99.</td><td>&nbsp;</td><td align="left" valign="top">Arif, S.M.; Holliday, J.D.; Willett, P. Inverse frequency weighting of fragments for similarity-based virtual screening. J. Chem. Inf. Model., 2010, 50, 1340-1349.</td></tr>
<tr><td align="right" valign="top">100.</td><td>&nbsp;</td><td align="left" valign="top">Chen, B.; Mueller, C.; Willett, P. Combinations rules for group fusion in similarity-based virtual screening. Mol. Inf. 2010, 29, 533-541.</td></tr>
<tr><td align="right" valign="top">101.</td><td>&nbsp;</td><td align="left" valign="top">Willett, P.; Similarity searching using 2D structural fingerprints. Chemoinformatics and Computational Chemical Biology. Methods in Molecular Biology. 2011, 672, 133-58.</td></tr>
<tr><td align="right" valign="top">102.</td><td>&nbsp;</td><td align="left" valign="top">Berglund, A.E.; Head, R.D. PZIM: A method for similarity searching using atom environments and 2d alignment. J. Chem. Inf. Model. 2010, 50, 1790-1795.</td></tr>
<tr><td align="right" valign="top">103.</td><td>&nbsp;</td><td align="left" valign="top">Baldi, P.; Nasr, R. When is chemical similarity significant? The statistical distribution of chemical similarity scores and its extreme values. J. Chem. Inf. Model. 2010, 50, 1205-1222.</td></tr>
<tr><td align="right" valign="top">104.</td><td>&nbsp;</td><td align="left" valign="top">Godden, J.W.; Stahura, F.L,; Bajorath, J. Anatomy of fingerprint search calculations on structurally diverse sets of active compounds. J. Chem. Inf. Model. 2005, 45, 1812-1819.</td></tr>
<tr><td align="right" valign="top">105.</td><td>&nbsp;</td><td align="left" valign="top">Geppert, H.; Horvath, T.; Gartner, T.; Wrobel, S.; Bajorath, J. Support-vector-machine-based ranking significantly improves the effectiveness of similarity searching using 2d fingerprints and multiple reference compounds. J. Chem. Inf. Model. 2008, 48, 742-746.</td></tr>
<tr><td align="right" valign="top">106.</td><td>&nbsp;</td><td align="left" valign="top">Wang, Y.; Geppert, H.; Bajorath, J. Shannon entropy-based fingerprint similarity search strategy. J. Chem. Inf. Model., 2009, 49, 1687-1691.</td></tr>
<tr><td align="right" valign="top">107.</td><td>&nbsp;</td><td align="left" valign="top">Nisius, B.; Bajorath, J. Molecular fingerprint recombination: Generating hybrid fingerprints for similarity searching from different fingerprint types. ChemMedChem. 2009, 4, 1859-1863.</td></tr>
<tr><td align="right" valign="top">108.</td><td>&nbsp;</td><td align="left" valign="top">Vogt, M.; Bajorath, J. Predicting the Performance of Fingerprint Similarity Searching. Chemoinformatics and Computational Chemical Biology. Methods in Molecular Biology. 2011, 672, 159-173.</td></tr>
<tr><td align="right" valign="top">109.</td><td>&nbsp;</td><td align="left" valign="top">Muchmore, S.W.; Debe, D.A.; Metz, J.T.; Brown, S.P.; Martin, Y. .; Hajduk, P. H. Application of belief theory to similarity data fusion for use in analog searching and lead hopping. J. Chem. Inf. Model. 2008, 48, 941-948.</td></tr>
<tr><td align="right" valign="top">110.</td><td>&nbsp;</td><td align="left" valign="top">Bender, A.; Jenkins, J.L.; Scheiber, J.; Sukuru, S.C.K.; Glick, M.; Davies, J. W. How similar are similarity searching methods? A principal component analysis of molecular descriptor space. J. Chem. Inf. Model. 2009, 49, 108-119.</td></tr>
<tr><td align="right" valign="top">111.</td><td>&nbsp;</td><td align="left" valign="top">Sastry, M.; Lowrie, J.F.; Dixon, S.L.; Sherman, W. Large-scale sstematic analysis of 2D fingerprint methods and parameters to improve virtual screening enrichments. J. Chem. Inf. Model. 2010, 50, 771-784.</td></tr>
<tr><td align="right" valign="top">112.</td><td>&nbsp;</td><td align="left" valign="top">Tiikkainen, P.; Markt, P.; Wolber, G.; Kirchmair, J.; Distinto, S.; Poso, A.; Kallioniemi. O. Critical comparison of virtual screening methods against the MUV data set. J. Chem. Inf. Model., 2009, 49, 2168-2178.</td></tr>
<tr><td align="right" valign="top">113.</td><td>&nbsp;</td><td align="left" valign="top">Venkatraman, V.; Prez-Nueno, V. I.; Mavridis L.; Ritchie, D.W. Comprehensive comparison of ligand-based virtual screening tools against the DUD data set reveals limitations of current 3D methods. J. Chem. Inf. Model., 2010, 50,  2079-2093.</td></tr>
<tr><td align="right" valign="top">114.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://code.google.com/p/chem-fingerprints/" title="Visit http://code.google.com/p/chem-fingerprints/">Chemfp - Cheminformatics fingerprints file formats and tools.</a></td></tr>
<tr><td align="right" valign="top">115.</td><td>&nbsp;</td><td align="left" valign="top">Yan, A.; Gasteiger, J.; Prediction of aqueous solubility of organic compounds by topological descriptors. QSAR Comb Sci. 2003, 22, 821-829.</td></tr>
<tr><td align="right" valign="top">116.</td><td>&nbsp;</td><td align="left" valign="top">Lovering, F.; Bikker, J.; Humblet, C. Escape from flatland: Increasing saturation as an approach to improving clinical success. J. Med. Chem. 2009, 52, 6752-6756.</td></tr>
<tr><td align="right" valign="top">117.</td><td>&nbsp;</td><td align="left" valign="top">Hann, M.M.; Leach, A.R.; Harper, G. Molecular complexity and its impact on the probability of finding leads for drug discovery. J. Chem. Inf. Comput. Sci. 2001, 41, 856-864.</td></tr>
<tr><td align="right" valign="top">118.</td><td>&nbsp;</td><td align="left" valign="top">Schuffenhauer, S.; Brown, N.; Selzer, P.; Ertl, P.; Jacoby, E. Relationships between molecular complexity, biological activity, and structural diversity. J. Chem. Inf. Model., 2006, 46, 525-535.</td></tr>
<tr><td align="right" valign="top">119.</td><td>&nbsp;</td><td align="left" valign="top">Walters, W.P.; Green, J.; Weiss, J.R.; Murcko, M. A. What do medicinal chemists actually make? A 50-year retrospective. J. Med. Chem. 2011, 54, 6405-6416.</td></tr>
<tr><td align="right" valign="top">120.</td><td>&nbsp;</td><td align="left" valign="top">Park, S.K.; Miller, K.W. Random number generators: Good ones are hard to find. Communications of the ACM. 1998, 10, 1192- 1200.</td></tr>
<tr><td align="right" valign="top">121.</td><td>&nbsp;</td><td align="left" valign="top">Huang R.; Southall N.; Wang Y.; Yasgar A.; Shinn P.; Jadhav A.; Nguyen D. T.; Austin C. P. The NCGC pharmaceutical collection: A comprehensive resource of clinically approved drugs enabling repurposing and chemical genomics. Sci. Transl. Med. 2011, 80ps16.</td></tr>
<tr><td align="right" valign="top">122.</td><td>&nbsp;</td><td align="left" valign="top">Berman,  H. M.; Westbrook, J.; Feng, Z.; Gilliland G.; Bhat, T. N.; Weissig, H.; Shindyalov, I. N.; Bourne, P. E. The Protein Data Bank. Nucleic Acids Research. 2000, 28, 235-242.</td></tr>
<tr><td align="right" valign="top">123.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.jmol.org/" title="Visit http://www.jmol.org/">Jmol: An open-source Java viewer for chemical structures in 3D.</a></td></tr>
<tr><td align="right" valign="top">124.</td><td>&nbsp;</td><td align="left" valign="top">Lloyd, D. What is aromaticity? J. Chem. Inf. Comput. Sci. 1996, 36, 442-447.</td></tr>
<tr><td align="right" valign="top">125.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.rdkit.org/UGM/2012/Sayle_RDKitPerspective.pdf" title="Visit http://www.rdkit.org/UGM/2012/Sayle_RDKitPerspective.pdf">Sayle, R. Cheminformatics toolkits: A personal perspective.</a></td></tr>
<tr><td align="right" valign="top">126.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://hop.perl.plover.com/" title="Visit http://hop.perl.plover.com/">Dominus, M. J. Higher-order Perl.</a></td></tr>
<tr><td align="right" valign="top">127.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.opensmiles.org/opensmiles.pdf" title="Visit http://www.opensmiles.org/opensmiles.pdf">OpenSMILES.</a></td></tr>
<tr><td align="right" valign="top">128.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.moldb.net/opensmarts/" title="Visit http://www.moldb.net/opensmarts/">Vandermeersch, T. OpenSMARTS.</a></td></tr>
<tr><td align="right" valign="top">129.</td><td>&nbsp;</td><td align="left" valign="top">Riniker, S.; Landrum, G. A. Better informed distance geometry: Using what we know to improve conformation generation. JCIM. 2015, 55, 2562-2574.</td></tr>
<tr><td align="right" valign="top">130.</td><td>&nbsp;</td><td align="left" valign="top">Baell, J. B.; Holloway, G. A. New substructure filters for removal of Pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays. JMC. 2010, 53 2719-2740.</td></tr>
<tr><td align="right" valign="top">131.</td><td>&nbsp;</td><td align="left" valign="top">Baell, J. B.;  Nissink, J. W. M.  Seven year itch: Pan-assay interference compounds (PAINS) in 2017 - Utility and limitations. ACS Chem. Biol. 2018, 13, 36-44.</td></tr>
<tr><td align="right" valign="top">132.</td><td>&nbsp;</td><td align="left" valign="top">Tosco, P.; Balle, T.; Shiri F. Open3DALIGN: an open-source software aimed at unsupervised ligand alignment. JCAMD. 2011, 25, 777-783.</td></tr>
<tr><td align="right" valign="top">133.</td><td>&nbsp;</td><td align="left" valign="top">Bemis, G. W.; Murcko, M. A.  The properties of known drugs. 1. Molecular frameworks. J. Med. Chem. 1996, 39, 2887-2893.</td></tr>
<tr><td align="right" valign="top">134.</td><td>&nbsp;</td><td align="left" valign="top">Hartenfeller, M.; Eberle, M; Meier, P.;Nieto-Oberhuber,C.; Altmann, K.H. A collection of robust organic synthesis reactions for In silico molecule design. J. Chem. Inf. Model. 2011, 51, 3093-3098.</td></tr>
<tr><td align="right" valign="top">135.</td><td>&nbsp;</td><td align="left" valign="top">Ashton, M., Barnard. J.; Casset, F.; Charlton. M.; Downs, G.; Gorse, D.; Holliday, J.; Lahana, R.; Willett, P. Identification of diverse database subsets using property-based and fragment-based molecular descriptors, Quant. Struct.-Act. Relat., 2002. 598-604.</td></tr>
<tr><td align="right" valign="top">136.</td><td>&nbsp;</td><td align="left" valign="top">Butina, D. Unsupervised database clustering based on Daylight's fingerprint and Tanimoto similarity: A fast and automated way to cluster small and large data sets. J. Chem. Inf. Model. 1999, 39, 747-750.</td></tr>
<tr><td align="right" valign="top">137.</td><td>&nbsp;</td><td align="left" valign="top"><a href="https://drugdesigndata.org" title="Visit https://drugdesigndata.org">D3R: Drug design data resource.</a></td></tr>
<tr><td align="right" valign="top">138.</td><td>&nbsp;</td><td align="left" valign="top">Gasteiger, J.; Marseli,  M. Iterative equalization of oribital electronegatiity - A rapid access to atomic charges. Tetrahedron. 1980, 36, 3219-3228.</td></tr>
<tr><td align="right" valign="top">139.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://www.pymol.org" title="Visit http://www.pymol.org">PyMOL - A molecular visualization system on an open source foundation originally developed by Warren DeLano.</a></td></tr>
<tr><td align="right" valign="top">140.</td><td>&nbsp;</td><td align="left" valign="top">Hagemans D.; van Belzen I. A. E. M.; Moran Luengo T.; Rudiger S.G.D. A script to highlight hydrophobicity and charge on protein surfaces. Frontiers in Molecular Biosciences. 2015, 2, 56.</td></tr>
<tr><td align="right" valign="top">141.</td><td>&nbsp;</td><td align="left" valign="top">Jones S., Thornton, M.J. Analysis of protein-protein interaction sites using surface patches. J. Mol. Biol. 1997, 1-12.</td></tr>
<tr><td align="right" valign="top">142.</td><td>&nbsp;</td><td align="left" valign="top">Afsar Minhas, F. U. A.; Geiss, B. J.; Ben-Hur A. PAIRpred: Partner specific prediction of interacting residues from sequence and structure. Prot. Struct. Funct. Bioinf. 2013, 82, 1142-1155.</td></tr>
<tr><td align="right" valign="top">143.</td><td>&nbsp;</td><td align="left" valign="top">Xue, L. C.; Dobbs, D.; Bonvin, A. M. J. J.; Honavar, V. Computational prediction of protein interfaces: A review of data driven methods. FEBS Letters. 2015, 589, 3516-3526.</td></tr>
<tr><td align="right" valign="top">144.</td><td>&nbsp;</td><td align="left" valign="top">Lovell, S.C.; Davis, I. W.; Arendall, W. B.; de Bakker, P. I.; Word, J. M.; Prisant, M. G.; Richardson, J. S.; Richardson, D. C. Structure validation by Calpha geometry: phi,psi and Cbeta deviation. Proteins. 2003, 50, 437-450.</td></tr>
<tr><td align="right" valign="top">145.</td><td>&nbsp;</td><td align="left" valign="top"><a href="https://warwick.ac.uk/fac/sci/moac/people/students/peter_cock/r/ramachandran/" title="Visit https://warwick.ac.uk/fac/sci/moac/people/students/peter_cock/r/ramachandran/">An example of Ramachandran contour levels and colors.</a></td></tr>
<tr><td align="right" valign="top">146.</td><td>&nbsp;</td><td align="left" valign="top">Guba, W.; Meyder, A.; Rarey, M.; Hert, J. Torsion library reloaded: A new version of expert-derived SMARTS rules for assessing conformations of small molecules.  J. Chem. Inf. Model. 2016, 56. 1-5.</td></tr>
<tr><td align="right" valign="top">147.</td><td>&nbsp;</td><td align="left" valign="top">Pennington, L. D.; Aquila, B. M.; Choi, Y.;  Roman A. Valiulin, R. A,; Muegge, I. Positional Analogue Scanning: An effective strategy for multiparameter optimization in drug design. J. Med. Chem. 2020.</td></tr>
<tr><td align="right" valign="top">148.</td><td>&nbsp;</td><td align="left" valign="top"><a href="http://practicalcheminformatics.blogspot.com/2020/04/positional-analogue-scanning.html" title="Visit http://practicalcheminformatics.blogspot.com/2020/04/positional-analogue-scanning.html">Positional Analogue Scanning.</a></td></tr>
</table>
</div>
<br />
<center>
<img src="./images/h2o2.png">
</center>
</body>
</html>
